spec/models/file_set_spec.rb

```diff
@@ -205,93 +137,9 @@ class AltFile < ActiveFedora::Base
     end
   end
 
-  describe '#related_files' do
-    let!(:f1) { described_class.new }
-
-    context 'when there are no related files' do
-      it 'returns an empty array' do
-        expect(f1.related_files).to eq []
-      end
-    end
-
-    context 'when there are related files' do
-      let(:parent_work)   { FactoryGirl.create_for_repository(:work_with_files) }
-      let(:f1)            { parent_work.file_sets.first }
-      let(:f2)            { parent_work.file_sets.last }
-
-      subject { f1.reload.related_files }
-
-      it 'returns all files contained in parent work(s) but excludes itself' do
-        expect(subject).to include(f2)
-        expect(subject).not_to include(f1)
-      end
-    end
-  end
-
-  describe 'noid integration', :clean_repo do
-    let(:service) { instance_double(ActiveFedora::Noid::Service, mint: noid) }
-    let(:noid) { 'wd3763094' }
-    let!(:default) { Hyrax.config.enable_noids? }
-
-    before do
-      allow(ActiveFedora::Noid::Service).to receive(:new).and_return(service)
-    end
-
-    after { Hyrax.config.enable_noids = default }
-
-    context 'with noids enabled' do
-      before { Hyrax.config.enable_noids = true }
-
-      it 'uses the noid service' do
-        expect(service).to receive(:mint).once
-        subject.assign_id
-      end
-
-      context "after saving" do
-        let(:saved) { persister.save(resource: subject) }
-
-        it 'returns the expected identifier' do
-          expect(saved.id).to eq noid
-        end
-
-        it "has a treeified URL" do
-          expect(saved.uri.to_s).to end_with '/wd/37/63/09/wd3763094'
-        end
-      end
-
-      context 'when a url is provided' do
-        let(:url) { "#{ActiveFedora.fedora.host}/test/wd/37/63/09/wd3763094" }
-
-        it 'transforms the url into an id' do
-          expect(described_class.uri_to_id(url)).to eq 'wd3763094'
-        end
-      end
-    end
-
-    context 'with noids disabled' do
-      before { Hyrax.config.enable_noids = false }
-
-      it 'does not use the noid service' do
-        expect(service).not_to receive(:mint)
-        subject.assign_id
-      end
-    end
-  end
-
   context 'with access control metadata' do
     subject do
-      described_class.new do |m|
-        m.apply_depositor_metadata('jcoyne')
-        m.permissions_attributes = [{ type: 'person', access: 'read', name: 'person1' },
-                                    { type: 'person', access: 'read', name: 'person2' },
-                                    { type: 'group', access: 'read', name: 'group-6' },
-                                    { type: 'group', access: 'read', name: 'group-7' },
-                                    { type: 'group', access: 'edit', name: 'group-8' }]
-      end
-    end
-
-    it 'has read groups accessor' do
```

Comment:
Did you mean to remove this test?

Reply:
Yeah, this is a method on Valkyrie::Resource, not on this class.